llvm/tools/sycl-post-link/ModuleSplitter.cpp

```diff
@@ -629,6 +651,16 @@ void EntryPointGroup::rebuildFromNames(const std::vector<std::string> &Names,
   });
 }
 
+void EntryPointGroup::rebuild(const Module &M) {
```

Comment:
Where is this method used?

Reply:
It is a leftover from the previous version. Removed it.